### PR TITLE
Add Andrew Xu (ajx8) as contributor

### PR DESCRIPTION
**Pull request by andxu282**
### Summary 

This pull request adds my name to the Contributors list in the README.md document.

- [x] adds my name to the Contributors list

Remaining TODOs:

- [x] Check that my name is added on the README.md document

### Test Plan 

Needs to pass all CI checks, which it does.

### CODE REVIEW

**Comment by alexjamesgodfrey** [comment]
Well done!

File: README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ Then access http://localhost:8080/
 - **Will Spencer** - Developer
 - **Ben Shen** - Developer
 - **Noah Schiff** - Developer
+- **Andrew Xu** - Developer
```
Comment (einc) [inline README.md:35]: :)

**Review by alexjamesgodfrey — APPROVED** [review]
Well done. Nothing to criticize.

**Review by einc — APPROVED** [review]
woohoo first PR!

**Review by jessfeng — APPROVED** [review]
Nice! LGTM

**Pull request merged**